### PR TITLE
Genesis fix, working

### PR DESCRIPTION
**Pull request by xx979xx**

### CODE REVIEW

File: selfdrive/car/hyundai/carcontroller.py [hunk 1]
```diff
@@ -136,13 +136,13 @@ def update(self, sendcan, enabled, CS, frame, actuators, CamS):
           lkas11_byte3 = CamS.lkas11_b3
       # When LKAS Forward is disabled, we generate the entire LKAS message
     else:
-      lkas11_byte0 = self.lanes + 0x2
+      lkas11_byte0 = self.lanes + 0x06
```
Comment (emmertex) [inline selfdrive/car/hyundai/carcontroller.py:139]: Can you detail what this does, and why it's needed?

File: selfdrive/car/hyundai/carcontroller.py [hunk 2]
```diff
@@ -136,13 +136,13 @@ def update(self, sendcan, enabled, CS, frame, actuators, CamS):
...
       lkas11_byte1 = 0x00
       lkas11_byte2 = apply_steer_a
       lkas11_byte3 = apply_steer_b
       lkas11_byte4 = (self.idx * 16) + 0x04
       lkas11_byte5 = 0x00
-      lkas11_byte7 = 0x18
+      lkas11_byte7 = 0x02
```
Comment (emmertex) [inline selfdrive/car/hyundai/carcontroller.py:145]: Can you detail what this does, and why it's needed?

File: selfdrive/car/hyundai/carstate.py [hunk 3]
```diff
@@ -132,8 +132,8 @@ def update(self, cp):
 
     # There is a 2% error on Sorento GT due to slightly larger wheel diameter compared to poverty packs.
     #   Dash assumes about 4% which is excessive. Using 3% to be safe
-    if self.car_fingerprint == CAR.SORENTO:
-      self.v_wheel = self.v_wheel * 1.03 
+    if self.car_fingerprint == CAR.GENESIS:
+      self.v_wheel = self.v_wheel * 1.075
```
Comment (emmertex) [inline selfdrive/car/hyundai/carstate.py:136]: Have you checked this against GPS?
Your Dash could be reporting a false high speed (common for most manufacturers)
We want the speed to be accurate, not equal to the dash.

File: selfdrive/car/hyundai/carcontroller.py (hunk 2 above)
Comment (xx979xx) [reply selfdrive/car/hyundai/carcontroller.py:145]: cabana showed genesis lkas11_byte7 always=02, not sure about other hyundai cars

File: selfdrive/car/hyundai/carcontroller.py (hunk 1 above)
Comment (xx979xx) [reply selfdrive/car/hyundai/carcontroller.py:139]: this is to display LKAS icon on cluster display (white icon), which will make it easier to know if OP is working for not.

**Comment by emmertex** [comment]
All of these changes have been implemented into hyundai-dev and community
Thanks

**Pull request closed without merging**